core/templates/dev/head/components/rule_editor_directive.html

```diff
@@ -26,11 +26,6 @@
         </span>
       </span>
     </div>
-
-    <div ng-if="rule.type == 'FuzzyMatches'">
```

Comment:
Make sure to check for the 'FuzzyMatches' constant and delete it everywhere; ditto for RULE_TYPE_CLASSIFIER. Don't want to leave remnants lying around.

Reply:
It is not required in frontend anymore but we still need it in backend as I mentioned in previous comment.

Reply:
See comment above, we should fully remove or at least deprecate these constants.

Reply:
Removed it.